### PR TITLE
Add Axe Core v1.4.0.2

### PR DESCRIPTION
**Pull request by charlesrocket**
Add [Axe Core](https://github.com/AXErunners/axe) v1.4.0.2

New conditions after https://github.com/Homebrew/homebrew-cask/pull/55073

After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [ ] Checked the cask was not [already refused].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask/pulls
[already refused]: https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

File: Casks/axe-core.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+cask 'axe-core' do
+  version '1.4.0.2'
+  sha256 '70fb7ac30214ad6d66ae661c2dc5d8b520413d0aafd916d9065789e974c9f72b'
+
+  url "https://github.com/axerunners/axe/releases/download/v#{version}/axecore-#{version}-osx-unsigned.dmg"
+  appcast 'https://github.com/axerunners/axe/releases.atom'
+  name 'Axe'
+  homepage 'https://github.com/axerunners/axe'
+
+  app 'Axe-Qt.app'
```
Comment (vitorgalvao) [inline Casks/axe-core.rb]: [`target:` Should Only Be Used in Select Cases](https://github.com/homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/app.md#target-should-only-be-used-in-select-cases).

File: Casks/axe.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
+cask 'axe' do
```
Comment (vitorgalvao) [inline Casks/axe.rb]: Did not follow the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).

File: Casks/axe.rb [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  version '1.4.0.2'
+  sha256 '70fb7ac30214ad6d66ae661c2dc5d8b520413d0aafd916d9065789e974c9f72b'
+
+  url "https://github.com/axerunners/axe/releases/download/v#{version}/axecore-#{version}-osx-unsigned.dmg"
+  appcast 'https://github.com/axerunners/axe/releases.atom'
+  name 'Axe'
+  homepage 'https://github.com/axerunners/axe'
+
+  app 'Axe-Qt.app'
```
Comment (vitorgalvao) [inline Casks/axe.rb]: [`target:` Should Only Be Used in Select Cases](https://github.com/homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/app.md#target-should-only-be-used-in-select-cases).

File: Casks/axe.rb [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+  preflight do
+    set_permissions "#{staged_path}/Axe-Qt.app", '0755'
```
Comment (vitorgalvao) [inline Casks/axe.rb]: You seem to be one of the developers. Why is this necessary?

File: Casks/axe.rb (hunk 2 above)
Comment (charlesrocket) [reply Casks/axe.rb]: This was the output from `generate_cask_token`
```
Proposed token:               axe-core
Proposed file name:           axe-core.rb
Cask Header Line:             cask 'axe-core' do
```

File: Casks/axe.rb (hunk 3 above)
Comment (charlesrocket) [reply Casks/axe.rb]: App title is `Axe Core`

File: Casks/axe.rb (hunk 4 above)
Comment (charlesrocket) [reply Casks/axe.rb]: To avoid permission denied issues

File: Casks/axe.rb (hunk 3 above)
Comment (vitorgalvao) [reply Casks/axe.rb]: Please stop saying “please revert”. Instead, read what I’m linking you to. Casks need to follow the rules we set.

Specifically, [add this comment](https://github.com/Homebrew/homebrew-cask/blob/d2a6b26df69fc28c4d84d6f5198b2b652c2f414d/Casks/devonthink-pro-office.rb#L16) which is what applies in this case. You may then readd the `target`.

File: Casks/axe.rb (hunk 2 above)
Comment (vitorgalvao) [reply Casks/axe.rb]: [After](https://github.com/Homebrew/homebrew-cask/pull/65160/files#r296473518), you may renamed to `axe-core` again.

File: Casks/axe.rb (hunk 4 above)
Comment (vitorgalvao) [reply Casks/axe.rb]: Let me rephrase: why doesn’t it ship with these permissions in the first place?

File: Casks/axe.rb (hunk 3 above)
Comment (charlesrocket) [reply Casks/axe.rb]: My bad, thought I was following guides, but looks like missed

File: Casks/axe.rb (hunk 4 above)
Comment (charlesrocket) [reply Casks/axe.rb]: It does
```
drwxr-xr-x  *** Axe-Qt.app
```
I wasn't sure if the issue is due constructor descriptions or env so decided to include

**Comment by vitorgalvao** [comment]
All done. Thank you for the fast turnaround.
